README.md

```diff
@@ -62,12 +62,20 @@ If you followed the previous steps you should already be inside your local clone
 5. Follow [these instructions](https://github.com/IBM/unity-sdk#getting-the-watson-sdk-and-adding-it-to-unity) to add the Watson Unity SDK downloaded in step 1 to the project.
 6. Follow [these instructions](https://github.com/IBM/unity-sdk#configuring-your-service-credentials) to add your Speech To Text and Conversation service credentials (located on [IBM Bluemix](https://console.ng.bluemix.net/)).
 7. Select `Advanced Mode` in the configuration window.
-8. Click `Add Variable` and name your new variable `ConversationV1_ID` then set its value to the Workspace ID of your Conversation workspace.
-    ![Variable Configuration Example](doc/source/images/add_variable.png?raw=true)
+8. Open the script vr-speech-sandbox-cardboard/SpeechSandbox/Assests/Scripts/VoiceSpawner.cs and put your workspace ID on line #34 in the Start() method.
  You can find your workspace ID by selecting the expansion menu on your conversation workspace and selecting `View details`.
     ![View Details Location](doc/source/images/workspace_details.png?raw=true)
 9. In the Unity editor project tab, select Assets->Scenes->MainGame->MainMenu and double click to load the scene.
 10. Press Play
+11. To Build an android .apk file and deploy to your phone, you can File -> Build Settings (Cntrl + Shift +B) and click Build. When prompted you can name your build and then move it to your phone. 
```

Comment:
s/Cntrl/Ctrl/  I think is more common.  Probably `CTRL` even if you do that consistently with other stuff too